docs/2____common_concepts.adoc

```diff
@@ -110,14 +102,8 @@ Depending on the needs, it may be necessary to dynamically provision the Bus Sim
 In the third variant of the communication architecture, the bus simulation is built directly into the respective importer.
 The supported bus features are analogous to the <<Composition-with-dedicated-Bus-Simulation-FMU, Composition with dedicated Bus Simulation FMU>> use case.
 The corresponding limitations regarding the behavior of the bus simulation are importer-specific.
-The following figure illustrates two FMUs, which are integrated by an importer that directly supports this standard and needs no further Bus Simulation FMU:
+The following figure illustrates two FMUs, which are integrated by an importer that directly supports this standard and needs no further Bus Simulation FMU.
 
 .Bus simulation by using an importer with internal bus simulation support.
 [#figure-bus-feature-integrated-fmu-simulator]
 image::architecture_bus_simulation_importer.svg[width=50%, align="center"]
-
-This communication architecture enables the realization of complex bus simulations.
```

Comment:
Shall we really remove "This communication architecture enables the realization of complex bus simulations.
By using proprietary extension mechanisms of an existing importer it may also be possible to integrate other, for example manufacturer-specific, formats into a bus simulation."?

Reply:
Then we should propably say it just like you say it in your comment - the sentence I deleted just stated the obvious and seemed kind of useless.